data/private

```diff
@@ -126,34 +126,52 @@ full:instant.arubanetworks.com
 full:setmeup.arubanetworks.com
 
 # ASUS router
+# Reference: https://www.asus.com/support/FAQ/1005263/
+full:asusrouter.com
 full:router.asus.com
+full:www.asusrouter.com
 
 # KDE
 networkcheck.kde.org
 
+# Netgear router
+# Reference: https://kb.netgear.com/27199/I-can-t-access-my-router-what-do-I-do
+# https://www.netgear.com/business/services/aplogincom/
+full:routerlogin.com
+full:www.routerlogin.com
+
 # Other router
 hiwifi.com
 leike.cc
-miwifi.com
 my.router
 peiluyou.com
 phicomm.me
 router.ctc
-routerlogin.com
 
-# QQ Loopback
+# Plex Media Server
+# Reference: https://support.plex.tv/articles/206225077-how-to-use-secure-server-connections/
+plex.direct
```

Comment:
why not `full:plex.direct`?
// the `.direct` does not look like a valid TLD for the internet, but only for now.

Reply:
![image](https://user-images.githubusercontent.com/37755859/227916285-31512e21-9841-4eb2-9a7e-b427521ed81c.png)

As shown in screenshot above, Plex uses individual subdomains under the `plex.direct` for each server instance, not this domain directly.